admin_scripts/os2displaypc/install_wm_keyboard.sh

```diff
@@ -3,12 +3,12 @@
 # DESCRIPTION
 #%    This script installs, sets up and enables a wm (bspwm)
 #%    and an on-screen keyboard (onboard).
-#%    Intended for OS2displayPC.
+#%    Intended for using OS2borgerPC Kiosk with OS2display.
 #-    email           mfm@magenta.dk
 
 # PREREQUISITES:
-# 1. OS2displayPC - Installer Chromium
-# 2. OS2displayPC - Autostart Chromium
+# 1. OS2display - Installer Chromium
+# 2. OS2display - Autostart Chromium
```

Comment:
Jeg tænker at script-kategorien også skal hedde OS2borgerPC Kiosk, eller? Også apropos ændringerne i dev.rst.
Det eneste script der er relateret til OS2display er vel auto_activate_chromium, som ikke nævnes her?

...og så skulle overskriften til sidst her måske være "Setting up for Kiosk"?:
https://github.com/OS2borgerPC/server_image/blob/master/docs/install_and_run.rst
...eller noget i den stil?

Reply:
Jeg tænkte bare på de tre scripts som specifikke for OS2display (med opsætning af Chromium), men du har jo ret, det er de netop ikke.

Reply:
Ok. `install_wm_keyboard` er vel også bare beregnet til OS2borgerPC Kiosk og ikke OS2display, men nærmere til noget med inputfelter, såsom Aula?